### PR TITLE
 [[ Bug 20399 ]] Fix missing launcher icon on Linux

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: engine/src/license.h [hunk 1]
```diff
@@ -126,4 +126,38 @@ inline bool MCStringFromLicenseClass(MCLicenseClass p_class, bool p_simplified,
     return false;
 }
 
+static const struct { const char *tag; MCLicenseClass value; } s_edition_map[] =
+{
+    { "community", kMCLicenseClassCommunity },
```
Comment (montegoulding) [inline engine/src/license.h]: @livecodeian my only comment on this is to wonder if we should just have a single struct for both APIs to iterate by just adding `edition_tag` or something to the other one. Just to keep the map between one string value and the other clear in the code.

Comment (montegoulding) [reply engine/src/license.h]: @livecodeian just poking you to see what your thoughts are on the above. If you don't think it's worthwhile let me know and I'll review this ok

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 65e18b5

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 65e18b5a2a5a2240b7d01d562bf08596f35a55ab

**Comment by livecode-vulcan** [comment]
😎 test success 65e18b5a2a5a2240b7d01d562bf08596f35a55ab

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/988)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1004)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/978)
* try-community-universal-ios-iphoneos11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.1/builds/51)
* try-community-universal-ios-iphonesimulator11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.1/builds/51)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/982)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1005)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/988)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1004)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/987)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/997)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1020)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/984)

**Pull request merged**